frontend/src/Components/ProductTypes/Products.tsx

```diff
@@ -36,6 +42,7 @@ const Products = () => {
           transformData={transformPaymentData}
         />
       )}
+      {products.includes("auth") && (
```

Comment:
```suggestion

```

Reply:
I think it is okay to leave the button and have it say, "product not available" if it is investments or income_verification, or what have you.  Otherwise it is missing for "transactions" or "identity" when is should be available.  Or we could do, if(includes ("auth" ||"transactions"|| "identity"). or !includes("investments"||"income_verification"}. Either of these 3 optiions is fine with me.